analyzer/src/main/resources/init.gradle

```diff
@@ -242,10 +242,10 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
                             "Unknown id type: ${id.getClass().simpleName}".toString(), '', null)
                 }
             } else if (dependencyResult instanceof UnresolvedDependencyResult) {
-                return new DependencyImpl('', dependencyResult.attempted.displayName, '', '', '', [],
+                return new DependencyImpl('', dependencyResult.attempted.displayName.replace('project :', 'project _'), '', '', '', [],
```

Comment:
Should we maybe simply omit the `:`? AFAIR, in Gradle the `:` for projects is what `/` is for paths, i.e. a project starting with `:` is an "absolute" project, i.e. its name is evaluated towards the root project. Given that we use nothing but absolute projects here (right?), we probably could simply agree that projects are "absolute" even if they do not explicitly start with `:` and strip the leading `:`.

Reply:
I agree, done.